types/Cargo.toml

```diff
@@ -21,9 +21,14 @@ aptos-dkg = { workspace = true }
 aptos-experimental-runtimes = { workspace = true }
 aptos-infallible = { workspace = true }
 ark-bn254 = { workspace = true }
+ark-circom = { workspace = true }
```

Comment:
Is this actually used?

Reply:
Yes. This is used in `generate_keys_and_inputs` in `proof_simulation.rs`, an unused function intended to be used in the future to generate trapdoor+vk pairs for future versions of the circuit. To do this, we interface with circom.

Reply:
Just saw your higher level comment, not actually sure if we need the same circuit or not. Let me think about it

Reply:
Updated the PR description